docs/conda.rst

```diff
@@ -0,0 +1,275 @@
+installed on the system together with the other CUDA Toolkit components installed inside the Conda 
+environment.
+
+The :code:`environment.yml` file
+--------------------------------
+
+We prefer to specify as many dependencies as possible in the Conda :code:`environment.yml` file 
+and only specify dependencies in :code:`requirements.txt` for install via :code:`pip` that are not 
+available via Conda channels. Check the Horovod `installation guide`_ for details of required 
+dependencies.
+
+Channel Priority
+^^^^^^^^^^^^^^^^
+
+Use the recommended channel priorities. Note that :code:`conda-forge` has priority over 
+:code:`defaults` and :code:`pytorch` has priority over :code:`conda-forge`. ::
+
+    name: null
+
+    channels:
+    - pytorch
+    - conda-forge
+    - defaults
+
+Dependencies
+^^^^^^^^^^^^
+
+There are a few things worth noting about the dependencies.
+
+1. Even though you have installed the NVIDIA CUDA Toolkit manually you should still use Conda to 
+   manage the other required CUDA components such as :code:`cudnn` and :code:`nccl` (and the 
+   optional :code:`cupti`).
+2. Use two meta-packages, :code:`cxx-compiler` and :code:`nvcc_linux-64`, to make sure that 
+   suitable C, and C++ compilers are installed and that the resulting Conda environment is aware 
+   of the manually installed CUDA Toolkit.
+3. Horovod requires some controller library to coordinate work between the various Horovod 
+   processes. Typically this will be some MPI implementation such as `OpenMPI`_. However, rather 
+   than specifying the :code:`openmpi` package directly you should instead opt for `mpi4py`_ Conda 
+   package which provides a CUDA-aware build of OpenMPI.
+4. Horovod also support that `Gloo`_ collective communications library that can be used in place of 
+   MPI. Include :code:`cmake` in order to insure that the Horovod extensions for Gloo are built.
+
+Below are the core required dependencies. The complete :code:`environment.yml` file is available 
+on GitHub. ::
+
+    dependencies:
+    - bokeh=1.4
+    - cmake=3.16 # insures that Gloo library extensions will be built
+    - cudnn=7.6
+    - cupti=10.1
+    - cxx-compiler=1.0 # insures C and C++ compilers are available
+    - jupyterlab=1.2
+    - mpi4py=3.0 # installs cuda-aware openmpi
+    - nccl=2.5
+    - nodejs=13
+    - nvcc_linux-64=10.1 # configures environment to be "cuda-aware"
+    - pip=20.0
+    - pip:
+        - mxnet-cu101mkl==1.6.* # MXNET is installed prior to horovod
+        - -r file:requirements.txt
+    - python=3.7
+    - pytorch=1.5
+    - tensorboard=2.1
+    - tensorflow-gpu=2.1
+    - torchvision=0.6
+
+The :code:`requirements.txt` file
+---------------------------------
+
+The :code:`requirements.txt` file is where all of the :code:`pip` dependencies, including Horovod 
+itself, are listed for installation. In addition to Horovod we typically will also use :code:`pip` 
```

Comment:
I recommend using double backquotes (``) for code highlighting

Reply:
Done